generated-dockerfiles/readme.md

```diff
@@ -1,3 +0,0 @@
-# generated-dockerfiles
```

Comment:
Is there a reason you deleted this file? It shouldn't have been deleted.

Reply:
The file is re-added.  That was a mistake with the generate script.  I assumed everything in the folder would be generated.